components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/model/stats/TaskInfo.java

```diff
@@ -0,0 +1,39 @@
+package org.wso2.carbon.bpmn.rest.model.stats;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Created by natasha on 12/15/15.
```

Comment:
This default comments with author name is not allowed. Change it to a description of the class function.